### PR TITLE
changelog page changes pt.3

### PR DESCRIPTION
**Pull request by mchnjt**
Yep, here we go again... *sigh*

**What's new:**

• responsive, on-page tabs 
(might not display correctly on small screens, you have been warned)
• ~~implemented "about" button, as it is removed from the launcher ui~~
• ~~news tab :frpogr:~~
• we do a little trolling
• ~~some shitty archive tab~~

### CODE REVIEW

**Comment by mchnjt** [comment]
Update 2 changes
- Removed "about", "news", and "archives" button, but the text container is still on code
- Now has a photo header
- Use local image source, will be placed at `res` folder
- ~~i need perms to upload files~~

Plans:
- icons, wee

**Comment by mchnjt** [comment]
wow, i didn't thought about making a folder in my patch repo

**Comment by mchnjt** [comment]
PR now ready for review. I'm tired adding icons (yet)

File: changelog.html [hunk 1]
```diff
@@ -3,101 +3,93 @@
...
-	  - Now can run Minecraft 1.13 and above. Hoorah!<br>
-	  - Now can properly launch Forge and Fabric! To install one of these bad boys, just simply download their .jar files.<br>
+	  - Now can run Minecraft 1.13 and above<br>
+	  - Now can properly launch Forge and Fabric! To install one of these "modloaders", just simply download their .jar files.<br>
           - Multi-architecture support: ARMv7, aarch64, x86 and x86_64.<br>
-          - Now properly supports Custom Controls (either made by the Discord community or by you!)<br>
+          - <strike>Now properly supports Custom Controls (either made by the Discord community or by you!)</strike><br>
+          	  <b>——————</br>
+          <b>• What's New</b></br>
+          - Java 17 for running Minecraft Versions above 21w10a<br>
```
Comment (NotAHero04) [inline changelog.html:84]: The change in Java version happens in 21w19a bruh

Comment (GoneSubject) [reply changelog.html:84]: ._,

**Pull request merged**